docker/debian9/entrypoint.sh

```diff
@@ -81,6 +89,11 @@ function prepare() {
             g_start_args="--confPath $conf_path"
             ;;
         chunkserver)
+            if [ "$g_disk" ]; then
+                [[ ! -d $disk_mount_point ]] && die "disk mount point $disk_mount_point does not exist.\n"
+                mount -o rw,errors=remount-ro $g_disk $disk_mount_point
+                [[ $? -ne 0 ]] && die "mount disk device $g_disk to $disk_mount_point failed.\n"
+            fi
             g_binary="$g_prefix/sbin/curvebs-chunkserver"
             g_start_args="--conf=$conf_path"
             ;;
```

Comment:
1. Done.
2. The disk device will be mounted in container, and umounted from host by curveadm. It is guaranteed by curveadm.
3. Unnecessary for local disk device.
4. The service(chunkserver) container should exit if any errors encounted during disk mounting.
5. The is tracked by container's log.
6. Cannot make sure the mount command to be executed successfully, it may fail.